ddtrace/gateway/__init__.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
+
+
+class _Addresses(Enum):
```

Comment:
should we just move this `ddtrace.gateway` module to `ddtrace.internal.gateway`?

If this is all meant to be internal/private anyways

Reply:
good point - pushing this refactor

Reply:
Done (and reduced to a single file :) )